src/process/procctl.rs

```diff
@@ -0,0 +1,173 @@
+//! Bindings for the FreeBSD `procctl` system call.
```

Comment:
Could you add comments here and at the top of prctl.rs mentioning that these two files have some signatures and documentation in common? Hopefully that will help us keep them in sync if they evolve.

Reply:
Done.